### DISCUSSION TITLE
Meeting Guide Page Empty after accidentally disabling 12 Step Meeting List plugin

### DISCUSSION THREAD

**Discussion by Area18Webservant** [code4recovery/12-step-meeting-list#1199] [Q&A]
Hi all, 

Was having trouble with our new gravity forms plugin, and as one of the initial checks. I tried deactivating plugins one at a time to find any potential contamination. When I have done this before, I usually skip over the 12 step meeting guide plugin, but I accidentally disabled it this time. I enabled it again right away, but now the Meeting guide page is empty. How do I restore my meeting database on that page?

**Comment by joshreisner** [comment]
hm, try going to settings > permalinks > save changes?

**Reply by Area18Webservant** [reply]
First thing I tried. The link seems to have disappeared from our meeting page. How do I link to it again?

**Reply by joshreisner** [reply]
is this your site? https://idahoarea18aa.org/meetings

**Reply by joshreisner** [reply]
i can confirm that i just tried deactivating the plugin on our test site and the meetings page did go away, but was restored when i went to settings > permalinks > save changes

seems like a bug though, going to create a ticket to see if we can't prevent this behavior

**Reply by SIA-WEBSUP** [reply]
couldn't repro on my test site ...
/meetings page unavailable until I restore permaminks to 'Post name'
